bbc/rmp-shared-assets
# [PS 319] Branding colours for Radio1's Vintage popup station

## Jira
https://jira.dev.bbc.co.uk/browse/PS-319

## Testing
1. Go to http://www.test.bbc.co.uk/radio/player/bbc_radio_one (or any network radio player)
2. Inspect element and go to the the dom inspector and view the class on the body (for radio 1, there will be a class of ''bbc_radio_one".
3. Change this class to 'bbc_radio_one_vintage'
4. Observe the colours have changed to the Radio 1 Vintage
